hawkmoth/parser.py

```diff
@@ -385,7 +423,7 @@ def _get_args(cursor):
 
 def _function_fixup(cursor):
     """Parse additional details of a function declaration."""
-    args = _get_args(cursor)
+    args = _get_args(cursor, 'c')
```

Comment:
This "smells" a bit wrong, because `_function_fixup()` will also be called for C++. By reading the code I can see that it will work just fine with the 'c' domain, because the 'cpp' domain features within `_get_args()` are only relevant for methods etc.

But feels like the domain should be an invariant (apart from `extern "C"`) that you can rely on.

Reply:
Oh, missed this one, the notification went straight to the junk folder.

Yeah, you're right. Don't merge it yet then, I'll force push a fix for this later.

Reply:
I suggested merging *after* I'd commented on this one. :)

Reply:
Yeah, but this one made me feel particularly dirty xD

It's also the sort of thing that has no immediate pressure for a fix, which I don't like.

Anyway, I force pushed this one thing only. Thanks for the patience!